spec/features/admin_can_add_a_task_to_supplier_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin users can' do
+  context 'when there is a supplier with tasks' do
+    before do
+      stub_govuk_bank_holidays_request
+      supplier = FactoryBot.create(:supplier, name: 'First Supplier')
+      framework1 = FactoryBot.create(:framework, name: 'Framework 1')
+      FactoryBot.create(:framework, name: 'Framework 2')
+      FactoryBot.create(:agreement, framework: framework1, supplier: supplier)
+      FactoryBot.create(:task, supplier: supplier, framework: framework1, period_month: 5, period_year: 2019)
```

Comment:
Do we need to create this task against the supplier for both specs? Should there be `context 'there is already a task for this period and month'` that does it?

I can read this (with a little effort) and it doesn't block anything, though, and everything passes the "is `develop` better with this on it` test, and honestly given the time we've got for this I can't tell if I'm bikeshedding, which probably means I am.